### DISCUSSION TITLE
Error message for importing dask.dataframe module

### DISCUSSION THREAD

**Discussion by cutepikachu-cn** [dask/dask#10678] [General]
import dask.dataframe as dd
import dask.array as da
import dask.bag as db

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[3], line 1
----> 1 import dask.dataframe as dd
      2 import dask.array as da
      3 import dask.bag as db

File D:\anaconda\Lib\site-packages\dask\dataframe\__init__.py:3
      1 try:
      2     from ..base import compute
----> 3     from . import backends, dispatch, rolling
      4     from .core import (
      5         DataFrame,
      6         Index,
   (...)
     12         to_timedelta,
     13     )
     14     from .groupby import Aggregation

File D:\anaconda\Lib\site-packages\dask\dataframe\backends.py:23
     20 from dask.sizeof import SimpleSizeof, sizeof
     22 from ..utils import is_arraylike, typename
---> 23 from .core import DataFrame, Index, Scalar, Series, _Frame
     24 from .dispatch import (
     25     categorical_dtype_dispatch,
     26     concat,
   (...)
     36     union_categoricals_dispatch,
     37 )
     38 from .extensions import make_array_nonempty, make_scalar

File D:\anaconda\Lib\site-packages\dask\dataframe\core.py:52
     50 from . import methods
     51 from ._compat import PANDAS_GT_140, PANDAS_GT_150
---> 52 from .accessor import DatetimeAccessor, StringAccessor
     53 from .categorical import CategoricalAccessor, categorize
     54 from .dispatch import (
     55     get_parallel_type,
     56     group_split_dispatch,
     57     hash_object_dispatch,
     58     meta_nonempty,
     59 )

File D:\anaconda\Lib\site-packages\dask\dataframe\accessor.py:112
    101     """Accessor object for datetimelike properties of the Series values.
    102 
    103     Examples
   (...)
    106     >>> s.dt.microsecond  # doctest: +SKIP
    107     """
    109     _accessor_name = "dt"
--> 112 class StringAccessor(Accessor):
    113     """Accessor object for string properties of the Series values.
    114 
    115     Examples
   (...)
    118     >>> s.str.lower()  # doctest: +SKIP
    119     """
    121     _accessor_name = "str"

File D:\anaconda\Lib\site-packages\dask\dataframe\accessor.py:124, in StringAccessor()
    121 _accessor_name = "str"
    122 _not_implemented = {"get_dummies"}
--> 124 @derived_from(pd.core.strings.StringMethods)
    125 def split(self, pat=None, n=-1, expand=False):
    126     if expand:
    127         if n == -1:

AttributeError: module 'pandas.core.strings' has no attribute 'StringMethods'

**Comment by jrbourbeau** [comment]
I can't be sure what's going on here without more information. What version of `dask` and `pandas` are you using? My guess is the `pandas` version is older than the minimum supported version
